src/patternfly/components/Drawer/drawer.scss

```diff
@@ -218,10 +359,56 @@ $pf-c-drawer__panel--width-list: (25, 33, 50, 66, 75);
   }
 
   @include pf-apply-breakpoint($breakpoint, $pf-c-drawer--breakpoint-map) {
-    @each $width-value in $pf-c-drawer__panel--width-list {
-      .pf-c-drawer__panel.pf-m-width-#{$width-value}#{$breakpoint-name} {
-        --pf-c-drawer__panel--FlexBasis: #{percentage($width-value / 100)};
+    // Drawer inline
+    .pf-c-drawer.pf-m-inline#{$breakpoint-name} {
```

Comment:
I wonder, if the CSS is the same between `pf-m-inline` and `pf-m-static`, if we could either include `pf-m-static` in this loop as a selector if the breakpoint is "base", or maybe make a mixin for the common styles.